pom.xml

```diff
@@ -246,6 +246,19 @@
                 <artifactId>sonar-maven-plugin</artifactId>
                 <version>${sonar-maven-plugin.version}</version>
             </plugin>
+            <plugin>
```

Comment:
This is a duplicate dependency off line 223. The one above is just missing the config.

Reply:
Resolved in latest commit